deployments/liqo/templates/liqo-auth-deployment.yaml

```diff
@@ -67,6 +67,22 @@ spec:
           - "$(POD_NAMESPACE)"
           - "--resyncSeconds"
           - "30"
+          {{- if .Values.awsConfig.accessKeyId }}
+          - "--awsAccessKeyId"
+          - {{ .Values.awsConfig.accessKeyId | quote }}
+          {{- end }}
+          {{- if .Values.awsConfig.secretAccessKey }}
+          - "--awsSecretAccessKey"
```

Comment:
My 2 cents: I would avoid having a secret value exposed as a flag, but more preferably templated in a secret object dedicated to the identity.